.githooks/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
`set e`  and/or `set -x` here, without it this script won't prevent a commit being made, and it's output won't be visible.

also `/bin/sh`, there's no need to assume bash or it's location (iirc fedora installs it into `/usr/bin`)